src/pages/en/guides/markdown-content.md

```diff
@@ -178,113 +78,107 @@ For Markdown files, the `content` prop also has an `astro` property which holds
 }
 ```
 
-> Keep in mind that the only guaranteed properties coming from the `content` prop are `astro` and `url`.
+> 💡 `astro` and `url` are the only guaranteed properties provided by Astro in the `content` prop. The rest of the object is defined by your front matter variables.
 
-### Images and videos
+### Variables 
+
+Front matter variables can be used directly in your Markdown as properties of the `frontmatter` object.
+
+```markdown
+---
+layout: ../layouts/BaseLayout.astro
+author: Leon
+age: 42
+---
+# About the Author
 
-Using images or videos follows Astro’s normal import rules:
+{frontmatter.author} is {frontmatter.age} and lives in Toronto, Canada.
 
-- Place them in the `public/` as explained on the [project-structure page](/en/core-concepts/project-structure/#public)
-  - Example: Image is located at `/public/assets/img/astonaut.png` → Markdown: `![Astronaut](/assets/img/astronaut.png)`
-- Or use `import` as explained on the [imports page](/en/guides/imports#other-assets) (when using Astro’s Markdown Component)
+```
 
 ### Components
 
-You can import components from any framework into your markdown file and use it along the markdown syntax. Front matter variables are available to your components from the `frontmatter` object.
+You can import components into your Markdown file with `setup` and use them along with Markdown syntax. The `frontmatter` object is also available to any imported components.
```

Comment:
```suggestion
You can import components into your Markdown file with `setup` and use them alongside your Markdown content. The `frontmatter` object is also available to any imported components.
```

Reply:
I'll buy this. I was really thinking of it from a syntax/writing/mix-and-match standpoint, but I have no issues with content. With Markdown, the content *is* closely tied to the syntax.